plugins/fritz-box7490/de/fritzbox7490_

```diff
@@ -0,0 +1,191 @@
+                 ;;
+           DATAV)
+                 Echo graph_title  tägliches Daten-Volumen
+                 echo graph_vlabel Transfer in MB
+                 echo u.label      upload
+                 echo d.label      download
+                 echo g.label      gesamt
+                 ;; 
+           ES)
+                 echo graph_title  Sekunden mit Fehlern \(ES\)
+                 echo graph_vlabel Sekunden
+                 echo u.label      upload
+                 echo d.label      download
+                 ;; 
+           SES)
+                 echo graph_title  Sekunden mit vielen Fehlern \(SES\)
+                 echo graph_vlabel Sekunden
+                 echo u.label      upload
+                 echo d.label      download
+                 ;; 
+           CRC01)
+                 Echo graph_title  Nicht korrigierbare Fehler \(CRC\)
+                 echo graph_vlabel Anzahl
+                 echo u.label      upload
+                 echo d.label      download
+                 ;; 
+           CRC15)
+                 Echo graph_title  Nicht korrigierbare Fehler in 15 Min. \(CRC\)
+                 echo graph_vlabel Anzahl
+                 echo u.label      upload
+                 echo d.label      download
+                 ;; 
+           VERB)
+                 Echo graph_title  Anzahl der täglichen Wiederverbindung
+                 echo graph_vlabel Anzahl
+                 echo u.label      upload
+                 echo d.label      download
+                 ;; 
+           UPT)
+                 ;; 
+           *)
+                 echo "  ##############################################" >&2
+                 echo "  # Felerhafter Programmaufruf, zulässig sind: #" >&2
+                 echo "  # - fritzbox7490_DRATE                       #" >&2
+                 echo "  # - fritzbox7490_LLOSS                       #" >&2
+                 echo "  # - fritzbox7490_SNR                         #" >&2
+                 echo "  # - fritzbox7490_DATAV                       #" >&2
+                 echo "  # - fritzbox7490_CRC01                       #" >&2
+                 echo "  # - fritzbox7490_CRC15                       #" >&2
+                 echo "  # - fritzbox7490_ES                          #" >&2
+                 echo "  # - fritzbox7490_SES                         #" >&2
+                 echo "  # - fritzbox7490_VERB                        #" >&2
+#                echo "  # - fritzbox7490_UPT                         #" >&2
+                 echo "  ##############################################" >&2
+                 exit 2
+                 ;;
+       esac
+       exit 0
+  fi
+# Standard Config Section End ####
+
+# Wildcard-abhängige Datenauswertung
+# Wildcard-dependent data evaluation 
+  case $id in
+       LLOSS)
+            u_value=$(echo "${werte}" | awk 'match($0,/pfung OUT:.* ([0-9]+) db/,arr){print arr[1]};');               echo u.value $u_value
```

Comment:
Please use separate lines for the `echo foo.value` output - this is currently hard to read.